operator/cmd/mesh/operator-common.go

```diff
@@ -55,7 +55,7 @@ const (
 // isControllerInstalled reports whether an operator deployment exists in the given namespace.
 func isControllerInstalled(cs kubernetes.Interface, operatorNamespace string, revision string) (bool, error) {
 	orn := operatorResourceName
-	if revision != "" {
+	if revision != "" && revision != "default" {
```

Comment:
I don't think it's correct to do it here. https://github.com/istio/istio/blob/ae5fb1a659b3876ae66f1a0a790591ea0155a7f6/operator/cmd/mesh/operator-remove.go#L86-L88 These lines revise the empty revision to "default", which should be incorrect.

Reply:
Thanks. I made adjustments according to your commit：https://github.com/istio/istio/commit/0c8164f2dc0f7f405fd6036614ca8c11a4452f7e